internal/olm/manager.go

```diff
@@ -28,14 +28,16 @@ import (
 const (
 	DefaultVersion = "latest"
 	DefaultTimeout = time.Minute * 2
+	// DefaultOLMNamespace is the namespace where OLM is installed
+	DefaultOLMNamespace = "olm"
 )
 
 type Manager struct {
-	Client  *Client
-	Version string
-	Timeout time.Duration
-
-	once sync.Once
+	Client       *Client
+	Version      string
+	Timeout      time.Duration
+	OLMNamespace string
```

Comment:
Here could be just Namespace since it is the manager namespace. 
However,  it is fine. I do not think that you need to change too.